lib/chef/resource/aws_cache_subnet_group.rb

```diff
@@ -0,0 +1,22 @@
+require 'chef/provisioning/aws_driver/aws_resource'
+
+class Chef::Resource::AwsCacheSubnetGroup < Chef::Provisioning::AWSDriver::AWSResource
+  aws_sdk_type AWS::ElastiCache, id: :group_name
+
+  attribute :group_name, kind_of: String, name_attribute: true
+  attribute :description, kind_of: String, required: true
+  attribute :subnets,
+            kind_of: [ String, Array ],
```

Comment:
You don't need the `LookupOptions` you have in the provider unless you also add `AwsSubnet, AWS::EC2::Subnet` like in the [aws_network_interface](https://github.com/chef/chef-provisioning-aws/blob/master/lib/chef/resource/aws_network_interface.rb#L14)